### PR TITLE
retrieve css for fskdbview pre tag

### PR DESCRIPTION
**Pull request by ahmadswaid**
https://github.com/RakipInitiative/ModelRepository/issues/313

### CODE REVIEW

**Review by schuelet — APPROVED** [review]

**Comment by ahmadswaid** [comment]
@schuelet @miguelalba: it's not enouph to have the approval to merge.

**Comment by mfilter** [comment]
@schuelet : is the issue with the missing public key something that @miguelalba must take care of?

**Comment by schuelet** [comment]
@ahmadswaid I switched the distribution to Xenial (which is a newer Ubuntu) and that seems to work.  I am waiting until it is done and then create a PR for you to approve, then the issue should be gone for now.

**Pull request merged**
